metabrainz/payments/stripe/views.py

```diff
@@ -77,18 +82,24 @@ def pay():
         }
 
     try:
-        session = stripe.checkout.Session.create(**session_config)
+        session = stripe.checkout.Session.create(**session_config, api_key=api_key)
         return redirect(session.url, code=303)
     except Exception as e:
         current_app.logger.error(e, exc_info=True)
         return redirect(url_for("payments.error", is_donation=is_donation))
 
 
-@payments_stripe_bp.route("/webhook/", methods=["POST"])
-def webhook():
+@payments_stripe_bp.route("/webhook/<currency>/", methods=["POST"])
```

Comment:
We'll need to install a new webhook and change the other one at deploy time?

Reply:
Change it at deploy time.